version: "2"
services:
  qgisserver:
    restart: always
    image: gisquick/qgis-server
    volumes:
      - ./_data/publish:/publish/:ro
    ports:
      - "9000:90"

  django:
    restart: always
    image: gisquick/django:latest
    links:
      - qgisserver
    volumes:
      - ./_data/media:/var/www/gisquick/media/
      - ./_data/data:/var/www/gisquick/data/
      - ./_data/publish:/publish/
    environment:
      - GUNICORN_ERRORLOG=-
      - GUNICORN_ACCESSLOG=-

  nginx:
    restart: unless-stopped
    image: gisquick/nginx
    links:
      - django
    volumes_from:
      - django:ro
    volumes:
      - /etc/letsencrypt/:/etc/letsencrypt/
    ports:
      - "4433:443"
    environment:
      - NGINX_HOST=rain1.fsv.cvut.cz
